domains/dyikes.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "dyikes",
+        "email": "f@power.slmail.me"
+    },
+    "record": {
+        "A": "dyikes.synology.me"
```

Comment:
A records should be an array of IPs. If you're using a hostname that should be a CNAME record.